### ISSUE TITLE
build fails on OSX 11 M1

### ISSUE THREAD

**Issue by dcolley** [osxfuse/osxfuse#750]
Following the build instructions - but taking HEAD - as follows:

```
git clone --recursive  git://github.com/osxfuse/osxfuse.git osxfuse

cd osxfuse

./build.sh -t distribution
Assertion 'common_is_variable DEFAULT_SDK_11_0_ARCHITECURES' failed
at common_variable_clone (./build.d/lib/common.sh, line 501)
at common_variable_clone (./build.d/lib/common.sh, line 514)
at build_target_invoke (./build.sh, line 656)
at build_main (./build.sh, line 942)
at main (./build.sh, line 950)
Terminated: 15
```

I'm on Big Sur 11.0.1
I have Xcode Version 12.2 (12B45b) and command line tools installed.

```
pkgutil --pkg-info=com.apple.pkg.CLTools_Executables
package-id: com.apple.pkg.CLTools_Executables
version: 12.2.0.0.1.1604076827
volume: /
location: /
install-time: 1606428485
groups: com.apple.FindSystemFiles.pkg-group 
```

**Comment by dcolley** [comment]
I tried hacking the build.d / sh file.

```
diff --git a/build.d/defaults.sh b/build.d/defaults.sh
index 4d9c9d9..caa22cc 100755
--- a/build.d/defaults.sh
+++ b/build.d/defaults.sh
@@ -53,7 +53,10 @@ declare -r  DEFAULT_SDK_10_12_COMPILER="com.apple.compilers.llvm.clang.1_0"
 declare -ra DEFAULT_SDK_10_13_ARCHITECURES=("i386" "x86_64")
 declare -r  DEFAULT_SDK_10_13_COMPILER="com.apple.compilers.llvm.clang.1_0"
 
-declare -ra DEFAULT_SDK_SUPPORTED=("10.9" "10.10" "10.11" "10.12" "10.13")
+declare -ra DEFAULT_SDK_11_0_ARCHITECURES=("arm64" "x86_64")
+declare -r  DEFAULT_SDK_11_0_COMPILER="com.apple.compilers.llvm.clang.1_0"
+
+declare -ra DEFAULT_SDK_SUPPORTED=("10.9" "10.10" "10.11" "10.12" "10.13" "11.0")
 
 declare     DEFAULT_SDK="`macos_get_version`"
 declare -r  DEFAULT_BUILD_CONFIGURATION="Release"
```

... but I am not able to go any further

```
./build.sh -t distribution -v 9
DEFAULT_BUILD_CONFIGURATION=Release
DEFAULT_BUILD_DIRECTORY=/tmp/osxfuse
DEFAULT_LOG_VERBOSE=2
DEFAULT_PREFIX=''
DEFAULT_SDK=11.0
DEFAULT_SDK_10_10_ARCHITECURES=([0]=i386 [1]=x86_64)
DEFAULT_SDK_10_10_COMPILER=com.apple.compilers.llvm.clang.1_0
DEFAULT_SDK_10_11_ARCHITECURES=([0]=i386 [1]=x86_64)
DEFAULT_SDK_10_11_COMPILER=com.apple.compilers.llvm.clang.1_0
DEFAULT_SDK_10_12_ARCHITECURES=([0]=i386 [1]=x86_64)
DEFAULT_SDK_10_12_COMPILER=com.apple.compilers.llvm.clang.1_0
DEFAULT_SDK_10_13_ARCHITECURES=([0]=i386 [1]=x86_64)
DEFAULT_SDK_10_13_COMPILER=com.apple.compilers.llvm.clang.1_0
DEFAULT_SDK_10_9_ARCHITECURES=([0]=i386 [1]=x86_64)
DEFAULT_SDK_10_9_COMPILER=com.apple.compilers.llvm.clang.1_0
DEFAULT_SDK_11_0_ARCHITECURES=([0]=arm64 [1]=x86_64)
DEFAULT_SDK_11_0_COMPILER=com.apple.compilers.llvm.clang.1_0
DEFAULT_SDK_SUPPORTED=([0]=10.9 [1]=10.10 [2]=10.11 [3]=10.12 [4]=10.13 [5]=11.0)
DEFAULT_SOURCE_DIRECTORY=/Users/derek/Sites/osxfuse
BUILD_SOURCE_DIRECTORY=/Users/derek/Sites/osxfuse
BUILD_BUILD_DIRECTORY=/tmp/osxfuse
Search for Xcode
Xcode 12.2 found at path '/Applications/Xcode.app/Contents/Developer'
XCODE_12_2_PATH=/Applications/Xcode.app/Contents/Developer
XCODE_INSTALLED=([0]=12.2)
XCODE_SDK_11_0_XCODE=([0]=12.2)
XCODE_SDK_INSTALLED=([0]=11.0)
Done searching for Xcode
Source extension osxfuse
T:distribution       | Source target distribution
T:distribution       | Invoke action build
T:distribution       | BUILD_TARGET_OPTION_ARCHITECTURES=([0]=arm64 [1]=x86_64)
T:distribution       | BUILD_TARGET_OPTION_BUILD_CONFIGURATION=Release
T:distribution       | BUILD_TARGET_OPTION_BUILD_SETTINGS=()
T:distribution       | BUILD_TARGET_OPTION_CODE_SIGN_IDENTITY=''
T:distribution       | BUILD_TARGET_OPTION_DEBUG_DIRECTORY=''
T:distribution       | BUILD_TARGET_OPTION_DEPLOYMENT_TARGET=11.0
T:distribution       | BUILD_TARGET_OPTION_GROUP=''
T:distribution       | BUILD_TARGET_OPTION_MACROS=()
T:distribution       | BUILD_TARGET_OPTION_OWNER=''
T:distribution       | BUILD_TARGET_OPTION_PREFIX=''
T:distribution       | BUILD_TARGET_OPTION_PRODUCT_SIGN_IDENTITY=''
T:distribution       | BUILD_TARGET_OPTION_PROVISIONING_PROFILE_SPECIFIER=''
T:distribution       | BUILD_TARGET_OPTION_ROOT=0
T:distribution       | BUILD_TARGET_OPTION_SDK=11.0
T:distribution       | BUILD_TARGET_OPTION_XCODE=12.2
T:distribution       | DISTRIBUTION_KEXT_TASKS=([0]=11.0)
T:distribution       | Clean target
T:distribution       | Source target distribution
T:distribution       | Invoke action clean
T:distribution       | BUILD_TARGET_OPTION_ARCHITECTURES=([0]=arm64 [1]=x86_64)
T:distribution       | BUILD_TARGET_OPTION_BUILD_CONFIGURATION=Release
T:distribution       | BUILD_TARGET_OPTION_BUILD_SETTINGS=()
T:distribution       | BUILD_TARGET_OPTION_CODE_SIGN_IDENTITY=''
T:distribution       | BUILD_TARGET_OPTION_DEBUG_DIRECTORY=''
T:distribution       | BUILD_TARGET_OPTION_DEPLOYMENT_TARGET=11.0
T:distribution       | BUILD_TARGET_OPTION_GROUP=''
T:distribution       | BUILD_TARGET_OPTION_MACROS=()
T:distribution       | BUILD_TARGET_OPTION_OWNER=''
T:distribution       | BUILD_TARGET_OPTION_PREFIX=''
T:distribution       | BUILD_TARGET_OPTION_PRODUCT_SIGN_IDENTITY=''
T:distribution       | BUILD_TARGET_OPTION_PROVISIONING_PROFILE_SPECIFIER=''
T:distribution       | BUILD_TARGET_OPTION_ROOT=0
T:distribution       | BUILD_TARGET_OPTION_SDK=11.0
T:distribution       | BUILD_TARGET_OPTION_XCODE=12.2
T:distribution       | Removing target build directory '/tmp/osxfuse/distribution'
T:distribution       | Completed action clean
T:distribution       | Build target for macOS 11.0
T:fsbundle           | Source target fsbundle
T:fsbundle           | Invoke action build
T:fsbundle           | BUILD_TARGET_OPTION_ARCHITECTURES=([0]=arm64 [1]=x86_64)
T:fsbundle           | BUILD_TARGET_OPTION_BUILD_CONFIGURATION=Release
T:fsbundle           | BUILD_TARGET_OPTION_BUILD_SETTINGS=()
T:fsbundle           | BUILD_TARGET_OPTION_CODE_SIGN_IDENTITY=''
T:fsbundle           | BUILD_TARGET_OPTION_DEBUG_DIRECTORY=''
T:fsbundle           | BUILD_TARGET_OPTION_DEPLOYMENT_TARGET=11.0
T:fsbundle           | BUILD_TARGET_OPTION_GROUP=''
T:fsbundle           | BUILD_TARGET_OPTION_MACROS=()
T:fsbundle           | BUILD_TARGET_OPTION_OWNER=''
T:fsbundle           | BUILD_TARGET_OPTION_PREFIX=''
T:fsbundle           | BUILD_TARGET_OPTION_PRODUCT_SIGN_IDENTITY=''
T:fsbundle           | BUILD_TARGET_OPTION_PROVISIONING_PROFILE_SPECIFIER=''
T:fsbundle           | BUILD_TARGET_OPTION_ROOT=0
T:fsbundle           | BUILD_TARGET_OPTION_SDK=11.0
T:fsbundle           | BUILD_TARGET_OPTION_XCODE=12.2
T:fsbundle           | FSBUNDLE_KEXT_TASKS=([0]=11.0)
T:fsbundle           | Clean target
T:fsbundle           | Source target fsbundle
T:fsbundle           | Invoke action clean
T:fsbundle           | BUILD_TARGET_OPTION_ARCHITECTURES=([0]=arm64 [1]=x86_64)
T:fsbundle           | BUILD_TARGET_OPTION_BUILD_CONFIGURATION=Release
T:fsbundle           | BUILD_TARGET_OPTION_BUILD_SETTINGS=()
T:fsbundle           | BUILD_TARGET_OPTION_CODE_SIGN_IDENTITY=''
T:fsbundle           | BUILD_TARGET_OPTION_DEBUG_DIRECTORY=''
T:fsbundle           | BUILD_TARGET_OPTION_DEPLOYMENT_TARGET=11.0
T:fsbundle           | BUILD_TARGET_OPTION_GROUP=''
T:fsbundle           | BUILD_TARGET_OPTION_MACROS=()
T:fsbundle           | BUILD_TARGET_OPTION_OWNER=''
T:fsbundle           | BUILD_TARGET_OPTION_PREFIX=''
T:fsbundle           | BUILD_TARGET_OPTION_PRODUCT_SIGN_IDENTITY=''
T:fsbundle           | BUILD_TARGET_OPTION_PROVISIONING_PROFILE_SPECIFIER=''
T:fsbundle           | BUILD_TARGET_OPTION_ROOT=0
T:fsbundle           | BUILD_TARGET_OPTION_SDK=11.0
T:fsbundle           | BUILD_TARGET_OPTION_XCODE=12.2
T:fsbundle           | Removing target build directory '/tmp/osxfuse/fsbundle'
T:fsbundle           | Completed action clean
T:fsbundle           | Build target for macOS 11.0
Command line invocation:
    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -configuration Release CONFIGURATION_BUILD_DIR=/tmp/osxfuse/fsbundle SDKROOT=macosx11.0 "ARCHS=arm64 x86_64" GCC_VERSION=com.apple.compilers.llvm.clang.1_0 MACOSX_DEPLOYMENT_TARGET=11.0 CODE_SIGN_IDENTITY= PROVISIONING_PROFILE_SPECIFIER= -project osxfusefs.xcodeproj -target osxfuse.fs clean build

Build settings from command line:
    ARCHS = arm64 x86_64
    CODE_SIGN_IDENTITY = 
    CONFIGURATION_BUILD_DIR = /tmp/osxfuse/fsbundle
    GCC_VERSION = com.apple.compilers.llvm.clang.1_0
    MACOSX_DEPLOYMENT_TARGET = 11.0
    PROVISIONING_PROFILE_SPECIFIER = 
    SDKROOT = macosx11.0

note: Using new build system
note: Using codesigning identity override: 

** CLEAN SUCCEEDED **

note: Using new build system
note: Using codesigning identity override: 
note: Planning build
note: Constructing build description
CreateBuildDirectory /Users/derek/Sites/osxfuse/support/build (in target 'load_osxfuse' from project 'load_osxfuse')
    cd /Users/derek/Sites/osxfuse/support
    builtin-create-build-directory /Users/derek/Sites/osxfuse/support/build

warning: Traditional headermap style is no longer supported; please migrate to using separate headermaps and set 'ALWAYS_SEARCH_USER_PATHS' to NO. (in target 'mount_osxfuse' from project 'mount_osxfuse')
WriteAuxiliaryFile /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-own-target-headers.hmap (in target 'mount_osxfuse' from project 'mount_osxfuse')
    cd /Users/derek/Sites/osxfuse/support
    write-file /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-own-target-headers.hmap

WriteAuxiliaryFile /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-all-non-framework-target-headers.hmap (in target 'mount_osxfuse' from project 'mount_osxfuse')
    cd /Users/derek/Sites/osxfuse/support
    write-file /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-all-non-framework-target-headers.hmap

WriteAuxiliaryFile /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/all-product-headers.yaml (in target 'mount_osxfuse' from project 'mount_osxfuse')
    cd /Users/derek/Sites/osxfuse/support
    write-file /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/all-product-headers.yaml

WriteAuxiliaryFile /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-all-target-headers.hmap (in target 'mount_osxfuse' from project 'mount_osxfuse')
    cd /Users/derek/Sites/osxfuse/support
    write-file /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-all-target-headers.hmap

WriteAuxiliaryFile /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-generated-files.hmap (in target 'mount_osxfuse' from project 'mount_osxfuse')
    cd /Users/derek/Sites/osxfuse/support
    write-file /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-generated-files.hmap

WriteAuxiliaryFile /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse.hmap (in target 'mount_osxfuse' from project 'mount_osxfuse')
    cd /Users/derek/Sites/osxfuse/support
    write-file /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse.hmap

WriteAuxiliaryFile /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-project-headers.hmap (in target 'mount_osxfuse' from project 'mount_osxfuse')
    cd /Users/derek/Sites/osxfuse/support
    write-file /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-project-headers.hmap

CompileC /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/Objects-normal/x86_64/getmntopts.o /Users/derek/Sites/osxfuse/support/mount_osxfuse/getmntopts.c normal x86_64 c com.apple.compilers.llvm.clang.1_0.compiler (in target 'mount_osxfuse' from project 'mount_osxfuse')
    cd /Users/derek/Sites/osxfuse/support
    export LANG\=en_US.US-ASCII
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x c -target x86_64-apple-macos11.0 -fmessage-length\=202 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit\=0 -fcolor-diagnostics -std\=c99 -Wno-trigraphs -fpascal-strings -Os -Werror -Werror\=implicit-function-declaration -Wno-missing-field-initializers -Wmissing-prototypes -Wmissing-braces -Wparentheses -Wswitch -Wunused-function -Wunused-label -Wunused-parameter -Wunused-variable -Wunused-value -Wno-empty-body -Wuninitialized -Wunknown-pragmas -pedantic -Wshadow -Wfour-char-constants -Wno-conversion -Wno-constant-conversion -Wno-int-conversion -Wno-bool-conversion -Wno-enum-conversion -Wno-float-conversion -Wno-non-literal-null-conversion -Wno-objc-literal-conversion -Wsign-compare -Wshorten-64-to-32 -Wpointer-sign -Wnewline-eof -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk -fasm-blocks -fstrict-aliasing -Wdeprecated-declarations -fvisibility\=hidden -Wno-sign-conversion -Wno-infinite-recursion -Wno-comma -Wno-block-capture-autoreleasing -Wno-strict-prototypes -Wno-semicolon-before-method-body -iquote /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-generated-files.hmap -I/Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-own-target-headers.hmap -I/Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-all-target-headers.hmap -iquote /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-project-headers.hmap -I/tmp/osxfuse/fsbundle/include -I/Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/DerivedSources-normal/x86_64 -I/Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/DerivedSources/x86_64 -I/Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/DerivedSources -F/tmp/osxfuse/fsbundle -MMD -MT dependencies -MF /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/Objects-normal/x86_64/getmntopts.d --serialize-diagnostics /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/Objects-normal/x86_64/getmntopts.dia -c /Users/derek/Sites/osxfuse/support/mount_osxfuse/getmntopts.c -o /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/Objects-normal/x86_64/getmntopts.o

CompileC /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/Objects-normal/x86_64/mount_osxfuse.o /Users/derek/Sites/osxfuse/support/mount_osxfuse/mount_osxfuse.c normal x86_64 c com.apple.compilers.llvm.clang.1_0.compiler (in target 'mount_osxfuse' from project 'mount_osxfuse')
    cd /Users/derek/Sites/osxfuse/support
    export LANG\=en_US.US-ASCII
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x c -target x86_64-apple-macos11.0 -fmessage-length\=202 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit\=0 -fcolor-diagnostics -std\=c99 -Wno-trigraphs -fpascal-strings -Os -Werror -Werror\=implicit-function-declaration -Wno-missing-field-initializers -Wmissing-prototypes -Wmissing-braces -Wparentheses -Wswitch -Wunused-function -Wunused-label -Wunused-parameter -Wunused-variable -Wunused-value -Wno-empty-body -Wuninitialized -Wunknown-pragmas -pedantic -Wshadow -Wfour-char-constants -Wno-conversion -Wno-constant-conversion -Wno-int-conversion -Wno-bool-conversion -Wno-enum-conversion -Wno-float-conversion -Wno-non-literal-null-conversion -Wno-objc-literal-conversion -Wsign-compare -Wshorten-64-to-32 -Wpointer-sign -Wnewline-eof -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk -fasm-blocks -fstrict-aliasing -Wdeprecated-declarations -fvisibility\=hidden -Wno-sign-conversion -Wno-infinite-recursion -Wno-comma -Wno-block-capture-autoreleasing -Wno-strict-prototypes -Wno-semicolon-before-method-body -iquote /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-generated-files.hmap -I/Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-own-target-headers.hmap -I/Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-all-target-headers.hmap -iquote /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/mount_osxfuse-project-headers.hmap -I/tmp/osxfuse/fsbundle/include -I/Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/DerivedSources-normal/x86_64 -I/Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/DerivedSources/x86_64 -I/Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/DerivedSources -F/tmp/osxfuse/fsbundle -MMD -MT dependencies -MF /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/Objects-normal/x86_64/mount_osxfuse.d --serialize-diagnostics /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/Objects-normal/x86_64/mount_osxfuse.dia -c /Users/derek/Sites/osxfuse/support/mount_osxfuse/mount_osxfuse.c -o /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/Objects-normal/x86_64/mount_osxfuse.o
/Users/derek/Sites/osxfuse/support/mount_osxfuse/mount_osxfuse.c:40:10: error: 'fuse_ioctl.h' file not found with <angled> include; use "quotes" instead
#include <fuse_ioctl.h>
         ^~~~~~~~~~~~~~
         "fuse_ioctl.h"
/Users/derek/Sites/osxfuse/support/mount_osxfuse/mount_osxfuse.c:41:10: error: 'fuse_mount.h' file not found with <angled> include; use "quotes" instead
#include <fuse_mount.h>
         ^~~~~~~~~~~~~~
         "fuse_mount.h"
In file included from /Users/derek/Sites/osxfuse/support/mount_osxfuse/mount_osxfuse.c:41:
/Users/derek/Sites/osxfuse/common/fuse_mount.h:9:10: error: 'fuse_param.h' file not found with <angled> include; use "quotes" instead
#include <fuse_param.h>
         ^~~~~~~~~~~~~~
         "fuse_param.h"
In file included from /Users/derek/Sites/osxfuse/support/mount_osxfuse/mount_osxfuse.c:41:
In file included from /Users/derek/Sites/osxfuse/common/fuse_mount.h:9:
/Users/derek/Sites/osxfuse/common/fuse_param.h:11:10: error: 'fuse_preprocessor.h' file not found with <angled> include; use "quotes" instead
#include <fuse_preprocessor.h>
         ^~~~~~~~~~~~~~~~~~~~~
         "fuse_preprocessor.h"
/Users/derek/Sites/osxfuse/common/fuse_param.h:12:10: error: 'fuse_version.h' file not found with <angled> include; use "quotes" instead
#include <fuse_version.h>
         ^~~~~~~~~~~~~~~~
         "fuse_version.h"
In file included from /Users/derek/Sites/osxfuse/support/mount_osxfuse/mount_osxfuse.c:41:
In file included from /Users/derek/Sites/osxfuse/common/fuse_mount.h:9:
In file included from /Users/derek/Sites/osxfuse/common/fuse_param.h:12:
/Users/derek/Sites/osxfuse/common/fuse_version.h:11:10: error: 'fuse_preprocessor.h' file not found with <angled> include; use "quotes" instead
#include <fuse_preprocessor.h>
         ^~~~~~~~~~~~~~~~~~~~~
         "fuse_preprocessor.h"
In file included from /Users/derek/Sites/osxfuse/support/mount_osxfuse/mount_osxfuse.c:41:
/Users/derek/Sites/osxfuse/common/fuse_mount.h:10:10: error: 'fuse_version.h' file not found with <angled> include; use "quotes" instead
#include <fuse_version.h>
         ^~~~~~~~~~~~~~~~
         "fuse_version.h"
/Users/derek/Sites/osxfuse/support/mount_osxfuse/mount_osxfuse.c:42:10: error: 'fuse_param.h' file not found with <angled> include; use "quotes" instead
#include <fuse_param.h>
         ^~~~~~~~~~~~~~
         "fuse_param.h"
/Users/derek/Sites/osxfuse/support/mount_osxfuse/mount_osxfuse.c:43:10: error: 'fuse_version.h' file not found with <angled> include; use "quotes" instead
#include <fuse_version.h>
         ^~~~~~~~~~~~~~~~
         "fuse_version.h"
/Users/derek/Sites/osxfuse/support/mount_osxfuse/mount_osxfuse.c:345:35: error: initializer element is not a compile-time constant
static uintptr_t iosize         = FUSE_DEFAULT_IOSIZE;
                                  ^~~~~~~~~~~~~~~~~~~
In file included from /Users/derek/Sites/osxfuse/support/mount_osxfuse/mount_osxfuse.c:41:
In file included from /Users/derek/Sites/osxfuse/common/fuse_mount.h:9:
/Users/derek/Sites/osxfuse/common/fuse_param.h:135:53: note: expanded from macro 'FUSE_DEFAULT_IOSIZE'
#define FUSE_DEFAULT_IOSIZE                     (16 * PAGE_SIZE)
                                                 ~~~^~~~~~~~~~~
/Users/derek/Sites/osxfuse/support/mount_osxfuse/mount_osxfuse.c:391:9: error: initializer element is not a compile-time constant
        (void *)FUSE_DEFAULT_IOSIZE,
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
11 errors generated.

WriteAuxiliaryFile /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/Objects-normal/x86_64/mount_osxfuse.LinkFileList (in target 'mount_osxfuse' from project 'mount_osxfuse')
    cd /Users/derek/Sites/osxfuse/support
    write-file /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/Objects-normal/x86_64/mount_osxfuse.LinkFileList

** BUILD FAILED **


The following build commands failed:
	CompileC /Users/derek/Sites/osxfuse/support/build/mount_osxfuse.build/Release/mount_osxfuse.build/Objects-normal/x86_64/mount_osxfuse.o /Users/derek/Sites/osxfuse/support/mount_osxfuse/mount_osxfuse.c normal x86_64 c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
T:fsbundle           | Failed to build target
Terminated: 15
Received signal: SIGTERM
```

**Comment by bfleischer** [comment]
The source code that is available in the repository is for version 3.8.3, which supports macOS 10.5 to macOS 10.14. macOS 11 and Apple Silicon are not supported by version 3.8.3. You will need macFUSE 4 for your M1 Mac. The source code of version 4 is not publicly available.

**Comment by Wilkolicious** [comment]
@bfleischer Appreciate your work on osxfuse/macfuse!

Will the source for version 4 be open sourced in the future?

**Comment by dcolley** [comment]
My immediate need has faded... I switched from VeraCrypt to **_encrypted DMG_** for the time being.
I will check back from time to time to see when I can switch back.

**Comment by Wilkolicious** [comment]
@dcolley What are you attempting to achieve?  If you wish to use VeraCrypt with macFUSE, then it works as of this build: https://github.com/veracrypt/VeraCrypt/issues/702#issuecomment-734507631

To clarify, it works with `VeraCrypt_1.24-Update8` + `macFUSE 4.0.3` albeit via Rosetta 2.  Performance seems great, but more use-cases are needed.

Or perhaps you're adamant on only using from source built binaries?

**Comment by dcolley** [comment]
@Wilkolicious thanks! That comment is 16 hours old and I did not see it. I tried it and it works for me.
I am a purist, so when the normal distro didn't work I tried to compile from source... in the end I figured I'd just settle for any solution and switch back when a VeraCrypt is available for M1.
Turns out I only had to wait a couple of hours ;)

I think I'll close this ticket. My original problem was not a defect anyway.
Thanks for your help!
